_data/rsha256/rsha256.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Rahul Shah 
+institution: University of California, Berkeley
+profile_pic: rsha256.png 
+quote: It’s kind of fun to do the impossible.
+github_user: rsha256 
+---
+
```

Comment:
You might want to get rid of this extra line at the end of the file, in case it may cause formatting issues.